server/routes/appointments/appointmentsController.ts

```diff
@@ -755,6 +755,18 @@ export default class AppointmentsController {
     )
   }
 
+  private async getCaseWorker(accessToken: string, username: string): Promise<AuthUserDetails | string> {
+    try {
+      return this.hmppsAuthService.getSPUserByUsername(accessToken, username)
+    } catch (e) {
+      const interventionsServiceError = e as InterventionsServiceError
+      if (interventionsServiceError.status === 404) {
```

Comment:
do we wanna log a warning or 'invalid assumption' when we hit 404 here? 

i'm guessing we never expect this to happen?

Reply:
Sure. I'll add a log here.

Reply:
done